src/openrct2-ui/UiContext.cpp

```diff
@@ -337,14 +337,15 @@ class UiContext final : public IUiContext
                     ContextQuit();
                     break;
                 case SDL_WINDOWEVENT:
-                    if (e.window.event == SDL_WINDOWEVENT_SIZE_CHANGED)
+                    if (e.window.event == SDL_WINDOWEVENT_RESIZED)
                     {
+                        LOG_VERBOSE("New Window size: %ux%u\n", e.window.data1, e.window.data2);
                         OnResize(e.window.data1, e.window.data2);
                     }
 
                     switch (e.window.event)
                     {
-                        case SDL_WINDOWEVENT_SIZE_CHANGED:
+                        case SDL_WINDOWEVENT_RESIZED:
```

Comment:
I don't see the need for this change. The docs on `SDL_WINDOWEVENT_RESIZED` state:

> window has been resized to data1xdata2; this event is always preceded by `SDL_WINDOWEVENT_SIZE_CHANGED`

So two events with the same meaning? Do you have any reason to use over the other?

Reply:
SDL_CreateTexture returned null when used with SDL_WINDOWEVENT_SIZE_CHANGED, the dimensions are correct, this solves the problem, why? no idea.

Reply:
Oh the docs say what `data1`, `data2` carries for the updated event, but nothing about those for `SDL_WINDOWEVENT_SIZE_CHANGED`. I think that is enough to approve the change.